extension-manifest-v2/app/panel/components/Settings/__tests__/__snapshots__/GeneralSettings.jsx.snap

```diff
@@ -72,17 +72,6 @@ exports[`app/panel/Settings/GeneralSettings.jsx Snapshot tests with react-test-r
           <div
             className="s-checkbox-label"
           >
-            <span
```

Comment:
The last updated view may still have value as it shows when the adblocker lists were last updated. If the bug was that it may be 1970 (Unix epoch 0), would it be a quickfix to initialize it as Date.now() on a fresh installation and keep the display?